content/docs/08-user-management/4-roles-permissions-and-project-scope.md

```diff
@@ -0,0 +1,359 @@
+---
+title: "Project Scope Permissions"
+metaTitle: "Project Roles"
+metaDescription: "The list of Global Project Roles under Project Scope"
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+
+# Global Project Scope
+
+The Global Project Scope holds a cluster of resources in a logical grouping to that specific project. Users and Teams with specific Roles can be associated with the Project(s) you create. Below is a list of Roles types within the Project Scope. 
+
+
+1. [Cloud Account Admin](/08-user-management/4-roles-permissions-and-project-scope#cloud-account-admin)
+2. [Cloud Account Editor](/08-user-management/4-roles-permissions-and-project-scope#cloud-account-editor)
+3. [Cloud Account Viewer](/08-user-management/4-roles-permissions-and-project-scope#cloud-account-viewer)
+4. [Cluster Admin](/08-user-management/4-roles-permissions-and-project-scope#cluster-admin)
+5. [Cluster Profile Admin](/08-user-management/4-roles-permissions-and-project-scope#cluster-profile-admin)
+6. [Cluster Profile Editor](/08-user-management/4-roles-permissions-and-project-scope#cluster-profile-editor)
+7. [Cluster Profile Viewer](/08-user-management/4-roles-permissions-and-project-scope#cluster-profile-viewer)
+8. [Cluster Viewer](/08-user-management/4-roles-permissions-and-project-scope#cluster-viewer)
+9. [Project Admin](/08-user-management/4-roles-permissions-and-project-scope#project-administrator)
+10. [Project Editor](/08-user-management/4-roles-permissions-and-project-scope#project-editor)
+11. [Project Viewer](/08-user-management/4-roles-permissions-and-project-scope#project-viewer)
+12. [Workspace Admin](/08-user-management/4-roles-permissions-and-project-scope#workspace-admin)
+13. [Workspace Operator](/08-user-management/4-roles-permissions-and-project-scope#workspace-operator)
+
+
+Each user is assigned Roles and Permissions to the scopes, resources, and components.  
+
+<br />
+<br />
+<br />
+
+# Project
+<br />
+<br />
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Admin" key="Project Admin">
+<br />
+
+## Project Administrator
+
+|                    |         |          |            | **Permissions** |            |            |             |             |            |
```

Comment:
<img width="954" alt="image" src="https://user-images.githubusercontent.com/49595451/172313363-20658525-c4bb-4998-bfdb-e2e680e5f198.png">

First Column can be named "Permissions" and all other columns can be give "operations"